.github/workflows/merge.yml

```diff
@@ -183,6 +183,15 @@ jobs:
         env:
           cmd_type: UNIT_TEST
 
+      - name: Blackduck Scan
+        uses: lumada-common-services/gh-composite-actions@1.2.0
+        env:
+          BlackDuck_Project_Version: ${{ steps.version.outputs.current-version }}
+          BlackDuck_Project_Name: "${{ inputs.sonar_project_key }}"
```

Comment:
Should Blackduck shared the same project key than Sonar?

Reply:
After confirming with Diego, the project key is "Pentaho", “Pentaho” for the main build and “Pentaho SHIMS” for the shims .kar files.